### PR TITLE
fix: style nits

### PR DESCRIPTION
**Pull request by mkenigs**
Fix typos and align with style guide.

Notes I took while we were creating https://github.com/flox/flox/issues/850

### CODE REVIEW

File: cli/flox/src/commands/environment.rs [hunk 1]
```diff
@@ -158,11 +158,11 @@ impl Edit {
 
                             Please `exit` the environment and run `flox activate` to see these changes."});
                 } else {
-                    println!("✅ environment successfully edited");
+                    println!("✅  Environment successfully updated.");
```
Comment (jennymahmoudi) [inline cli/flox/src/commands/environment.rs:161]: accidentally added an extra space here

File: cli/flox/src/commands/environment.rs [hunk 2]
```diff
@@ -1398,7 +1398,7 @@ impl Pull {
         floxhub_host: &Url,
     ) -> (String, String) {
         let mut start_message =
-            format!("⬇️ remote: pulling and building {env_ref} from {floxhub_host}");
+            format!("⬇️  Remote: pulling and building {env_ref} from {floxhub_host}");
```
Comment (jennymahmoudi) [inline cli/flox/src/commands/environment.rs:1401]: extra space here as well

File: cli/flox/src/commands/environment.rs (hunk 1 above)
Comment (jennymahmoudi) [reply cli/flox/src/commands/environment.rs:161]: or is this intentional to account for the icon?

Comment (mkenigs) [reply cli/flox/src/commands/environment.rs:161]: Yeah that's deliberate, we want that after every emoji

**Review by jennymahmoudi — APPROVED** [review]

**Review by jennymahmoudi — APPROVED** [review]

**Review by ghudgins — APPROVED** [review]

**Pull request merged**